packages/typespec-test/test/contoso/spec/client.tsp

```diff
@@ -0,0 +1,7 @@
+import "./main.tsp";
+import "@azure-tools/typespec-client-generator-core";
+
+using Azure.ClientGenerator.Core;
+
+@@access( Azure.Contoso.WidgetManager.MyModel, Access.public);
+@@usage( Azure.Contoso.WidgetManager.MyModel, Usage.input);
```

Comment:
@qiaozha should RLC respect usage and access in client.tsp? does we ignore it by design?